packages/app/src/Element/NoteCreator.tsx

```diff
@@ -232,6 +238,38 @@ export function NoteCreator() {
       dispatch(setPollOptions(copy));
     }
   }
+  
+  function renderRelayCustomisation() {
+    return (
+      <div>
+        {Object.keys(relays.item || {}).filter((el) => relays.item[el].write).map((r,i,a) => 
+          <div className="card flex">
+            <div className="flex f-col f-grow">
+              <div>
+                <FormattedMessage defaultMessage="{relay}" values={{ relay:r }} />
```

Comment:
Don't think there is any reason to wrap this in `FormattedMessage`, there is nothing to localize

Reply:
agreed